openml/flows/sklearn_converter.py

```diff
@@ -0,0 +1,466 @@
+from collections import OrderedDict, defaultdict
+import importlib
+import inspect
+import json
+import json.decoder
+import six
+import warnings
+import sys
+
+import numpy as np
+import scipy.stats.distributions
+import sklearn.base
+import sklearn.model_selection
+# Necessary to have signature available in python 2.7
+from sklearn.utils.fixes import signature
+
+from .flow import OpenMLFlow
+from ..exceptions import OpenMLRestrictionViolated
+
+MAXIMAL_FLOW_LENGTH = 1024
+
+
+if sys.version_info >= (3, 5):
+    from json.decoder import JSONDecodeError
+else:
+    JSONDecodeError = ValueError
+
+
+class SklearnToFlowConverter(object):
```

Comment:
why is this an object? Feels like it should be a submodule - which it already is. There is no state associated with the class, right?